prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html

```diff
@@ -1,6 +1,28 @@
 <div class="container">
   <app-banner></app-banner>
   <div class="row">
+    <app-alert *ngIf="loginCancelled"
+               type="info"
+               icon="error_outline">
+      <ng-container #alertTitle
+                    class="alert-title">
+        Login Cancelled
+      </ng-container>
+      <ng-container #alertContent
+                    class="alert-content">
+        <p>You have cancelled your login to PRIME</p>
+        <p>For assistance with PRIME, contact {{ primeSupportPhoneNumber | phone }} or
+          <a [attr.href]="'mailto:' + primeSupportEmail">{{ primeSupportEmail }}</a>
```

Comment:
So there is some mixed review but i thought we wanted PRIMESupport@gov.bc.ca, but i see on the bottom of the page it is not capitalized. So curious if this is the case.

Reply:
good point. now using the prime phone and prime email component to make them consistent throughout the app.